src/NumericalAlgorithms/Interpolation/ZeroCrossingPredictor.cpp

```diff
@@ -0,0 +1,22 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "NumericalAlgorithms/Interpolation/ZeroCrossingPredictor.hpp"
+#include "NumericalAlgorithms/Interpolation/LinearLeastSquares.hpp"
+
+#include "DataStructures/DataVector.hpp"
+
+#include <vector>
```

Comment:
std includes after including the corresponding header file, but before other spectre includes